services/connected_account.yaml

```diff
@@ -184,7 +161,7 @@ Resources:
         CloudTrailOwnerAccount: !Ref CloudTrailOwnerAccount
         AuditAccount: !Ref AuditAccount
         MasterPayerAccount: !Ref MasterPayerAccount
-        LegacyAccount: !Ref LegacyAccount
+        LegacyAccount: !Ref ResourceOwnerAccount
```

Comment:
I would Remove this and change the contract in account manager, a quick look at that suggests it's not hard, may not even be required, otherwise I worry this will have odd side/downstream effects.

Reply:
I'm worried about breaking changes one way or the other. I confirmed the Reactor doesn't use this.

Reply:
Account manager 😉 it's not called the reactor anymore.

The account manager should send a None downstream if it doesn't find a legacy role to validate in most cases except for one. But I would go hard code that here to make sure it happens https://github.com/Cloudzero/reactor-account-manager/blob/71bc368e62802f33ba4cbba536c2e81ba7f3ff0c/cz/lib/verify.py#L362

I would also fix some sketchy behavior in the manual link process here https://github.com/Cloudzero/reactor-account-manager/blob/d862bb532215185457febb899ded353db4b28ffa/cz/functions/link/handler.py#L44

And here https://github.com/Cloudzero/reactor-account-manager/blob/d862bb532215185457febb899ded353db4b28ffa/cz/functions/link/handler.py#L68

Manual link looks like it's using the legacy role in some way and that needs to probably be fixed?

Reply:
Ok 👍🏼